diagnostics_tools/include/diagnostic_tools/periodic_event_status.h

```diff
@@ -66,9 +66,26 @@ class PeriodicEventStatusParams final {
 
   const Diagnostic &stale_diagnostic() const { return stale_diagnostic_; }
 
+  PeriodicEventStatusParams &short_term_avg_window(double short_term_avg_window) {
+    short_term_avg_window_ = short_term_avg_window;
+    return *this;
+  }
+
+  double short_term_avg_window() const { return short_term_avg_window_; }
+
+  PeriodicEventStatusParams &long_term_avg_window(double long_term_avg_window)
+  {
+    long_term_avg_window_ = long_term_avg_window;
+    return *this;
+  }
+
+  double long_term_avg_window() const { return long_term_avg_window_; }
+
  private:
   double min_acceptable_period_{-1};
   double max_acceptable_period_{5};
+  double short_term_avg_window_{1};
+  double long_term_avg_window_{10000};
```

Comment:
@patapufate these members should be of `size_t` type.

Reply:
Fixed.